src/content/doc-surrealdb/integration/http.mdx

```diff
@@ -2456,3 +2460,57 @@ curl -X GET \
 </TabItem>
 
 </Tabs>
+
+## Custom endpoint at `/api/:ns/:db/:endpoint` {#custom}
+
+<Since v="v2.2.0" />
+
```

Comment:
```suggestion
> [!CAUTION]
> Currently, this is an experimental feature as such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this, set the `SURREAL_CAPS_ALLOW_EXPERIMENTAL` [environment variable](/docs/surrealdb/cli/start/#experimental-capabilities) to `define_api`.
```